src/promptflow-evals/promptflow/evals/evaluators/_content_safety/flow/evaluate_with_rai_service.py

```diff
@@ -1,16 +1,17 @@
 import re
 import time
 from typing import List
+from urllib.parse import urlparse
 
 import numpy as np
 import requests
 from azure.core.credentials import TokenCredential
 from azure.identity import DefaultAzureCredential
 from constants import RAIService, Tasks
 from utils import get_harm_severity_level
-from urllib.parse import urlparse
 
 from promptflow.core import tool
+from promptflow.evals._user_agent import USER_AGENT
```

Comment:
This line would cause problem for downloading and uploading, as the flow doesn't have dependency on evals package.

Reply:
Should I hardcode the user-agent to be promptflow-evals?

Reply:
or replace it with:
```
import importlib.metadata
version = importlib.metadata.version("promptflow-evals")
USER_AGENT="{}/{}".format("promptflow-evals", version)
```